frontend/src/pages/trucks/components/add-truck-form/libs/helper/get-truck-entity.helper.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  type TruckEntity,
+  type TruckFormModel,
+} from '~/packages/trucks/libs/types/types.js';
+
+function getTruckEntity(payload: TruckFormModel): Omit<TruckEntity, 'id'> {
+  return {
+    manufacturer: payload.manufacturer.value,
+    year: payload.year,
+    towType: payload.towType.value,
+    capacity: payload.capacity,
+    pricePerKm: payload.pricePerKm,
+    licensePlateNumber: payload.licensePlateNumber,
+  };
+}
```

Comment:
use es6 syntax 
also you can rewrite it
```suggestion
const { id, ...truckDetails } = payload;

  return {
    ...truckDetails,
    manufacturer: truckDetails.manufacturer.value,
    towType: truckDetails.towType.value
  };
```